solutions/swb-reference/integration-tests/tests/isolated/sshKeys/create.test.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import ClientSession from '../../../support/clientSession';
+import Setup from '../../../support/setup';
+import HttpError from '../../../support/utils/HttpError';
+import RandomTextGenerator from '../../../support/utils/randomTextGenerator';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+describe('cannot create SSH key', () => {
+  const setup: Setup = new Setup();
+  let adminSession: ClientSession;
+  let project: { id: string };
+  const randomTextGenerator = new RandomTextGenerator(setup.getSettings().get('runId'));
+
+  beforeAll(async () => {
```

Comment:
beforeEach

Reply:
beforeAll should be sufficient in this test setting

Reply:
We only want 1 new cost center and 1 new project currently for the entire test suite so this should be a `beforeAll`

Reply:
After discussing, we are going to try `beforeEach` to error on the side of caution